test/rules/always_put_control_body_on_new_line.dart

```diff
@@ -19,6 +19,13 @@ testIfElse() {
     return; // OK
   else return; // LINT
 
+  if (false) {
+  }
+  else if (false) { // OK
```

Comment:
@bwilkerson this rule is not compatible with `dartfmt`. For instance the following code:

```dart
if (animation.value >= 0.5)
  return childSecondHalf;
```

is formatted to the following by `dartfmt` 

```dart
if (animation.value >= 0.5) return childSecondHalf;
```

and this result is not accepted by the current Flutter SDK style.